src/main/java/frc/team4909/robot/subsystems/climber/commands/BothLiftDown.java

```diff
@@ -0,0 +1,33 @@
+package frc.team4909.robot.subsystems.climber.commands;
+
+import edu.wpi.first.wpilibj.command.Command;
+import frc.team4909.robot.Robot;
+import frc.team4909.robot.RobotConstants;
+
+public class BothLiftDown extends Command{
+
+    public BothLiftDown(){
+        requires(Robot.climberSubsystem);
+        requires(Robot.elevatorSubsystem);
+    }
+
+    public void execute() {
+        Robot.climberSubsystem.setStiltsClimbSpeed(RobotConstants.liftClimbSpeed);
+            int climberPos = Robot.climberSubsystem.getPosition();
+            int elevPos = (int) -(climberPos * (1.1/1.3));
+            Robot.elevatorSubsystem.setPosition(elevPos);
+
+            Robot.elevatorSubsystem.holdingPosition = elevPos;
+            Robot.climberSubsystem.holdingStiltsPosition = climberPos;     
+    }
+
+    public void end(){
+        Robot.climberSubsystem.setStiltsClimbSpeed(0);
+    }
+
```

Comment:
I suspect there should be an `interrupted` method that calls `end();`. If the operator presses a different button that uses one of the two subsystems `require`d by this command, it will steal the subystems without stopping the climb.

Reply:
When a command is interrupted, I believe that the end() method is still called and doesn't have to be explicitly called

Reply:
that is the default as per https://github.com/wpilibsuite/allwpilib/blob/876c6504714859eb7aa453e4c378b5bb78efc2a9/wpilibj/src/main/java/edu/wpi/first/wpilibj/command/Command.java#L308

might need an @Override header though